assets/src/application/ui/calendars/dateDisplay/CalendarDateSwitcher/style.scss

```diff
@@ -0,0 +1,33 @@
+.ee-calendar-date-range-wrapper {
+	display: flex;
+	flex-flow: column nowrap;
+	height: 100%;
+	justify-content: space-between;
+	min-height: calc(var(--ee-padding-micro) * 45);
+	min-width: calc(var(--ee-padding-micro) * 34);
+	padding: var(--ee-padding-tiny) var(--ee-padding-micro);
+	text-align: center;
+	vertical-align: bottom;
+	width: 100%;
+
+	.ee-calendar-date-range {
+		align-content: space-between;
+		display: flex;
+		flex-flow: column nowrap;
+		height: 100%;
+		justify-content: space-evenly;
+	}
+
+	@media only screen and (max-width: 480px) {
+		min-height: auto;
+		.ee-calendar-date-range {
+			align-items: center;
+			flex-flow: row nowrap;
+			justify-content: center;
+
+			&__divider {
+				padding: 0 var(--ee-padding-big);
+			}
+		}
+	}
+}
```

Comment:
why is the css from CalendarDateRange copied into this component?!!?!
it's also not even loaded?!?

hmmm... did you copy the CalendarDateRange folder and then forget to delete this? 🤔

Reply:
yup, just removed